### PR TITLE
Put numpy instance frontend

### PR DESCRIPTION
**Pull request by IIDCII**
Close https://github.com/unifyai/ivy/issues/18530

### CODE REVIEW

**Comment by IIDCII** [comment]
Close #18530

**Comment by IIDCII** [comment]
Close #18530

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Review by AryanSharma21 — COMMENTED** [review]
Hey @IIDCII , you should add the closing tag in PR description and not in comments 🙂

File: ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py
Comment (CatB1t) [inline ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py:1]: Hello @IIDCII, why are those changes required? can you explain the reason for those modifications? Thanks!

File: ivy_tests/array_api_testing/test_array_api
Comment (CatB1t) [inline ivy_tests/array_api_testing/test_array_api:1]: Can we revert this?

**Review by CatB1t — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**